CHANGELOG.md

```diff
@@ -18,5 +18,7 @@
 - Removed unsused files and examples with make template_init #6
 - Updated README files #2
 - Added port and host to output.tf #13
+- Update vagrant-box, `version = ">= 0.9, < 0.10"` #17
```

Comment:
I suggest to move this sentence to `[0.1.1 UNRELEASED]`[-section](https://github.com/hannemariavister/terraform-nomad-nifiregistry/blob/master/CHANGELOG.md#011-unreleased), since version `[0.1.0]` has already been released.

Reply:
done